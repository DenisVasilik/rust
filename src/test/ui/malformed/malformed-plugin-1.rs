#![feature(plugin)]
#![plugin] //~ ERROR attribute must be of the form `#[plugin(...)]`

fn main() {}
