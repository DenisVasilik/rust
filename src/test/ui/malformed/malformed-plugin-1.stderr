error: attribute must be of the form `#[plugin(...)]`
  --> $DIR/malformed-plugin-1.rs:2:1
   |
LL | #![plugin] //~ ERROR attribute must be of the form `#[plugin(...)]`
   | ^^^^^^^^^^

error: aborting due to previous error

